debian/build.go

```diff
@@ -330,10 +326,10 @@ func main() {
 			Distros: serverDistros,
 			Versions: []version{
 				{
-					Version:             "1.6.0-alpha.0.2074-a092d8e0f95f52",
-					Revision:            "00",
-					Channel:             ChannelStable,
-					GetDownloadLinkBase: getCIBuildsDownloadLinkBaseFromVersion,
+					Version:          "1.6.0-alpha.0.2074-a092d8e0f95f52",
+					Revision:         "00",
+					Channel:          ChannelStable,
+					DownloadLinkBase: "https://dl.k8s.io/ci-cross/v1.6.0-alpha.0.2074+a092d8e0f95f52",
```

Comment:
@ixdy I did that initially, but it was just a mess to try to convert from the deb (non-semver) version to the semver-compliant k8s one. This is easiest. Once v1.6 is released, this will be removed.

It won't change and come out of sync fortunately.